mscore/palette.cpp

```diff
@@ -551,6 +551,39 @@ void Palette::applyPaletteElement(Element* element, Qt::KeyboardModifiers modifi
                   spanner->styleChanged();
                   score->cmdAddSpanner(spanner, idx, startSegment, endSegment);
                   }
+            else if (element->isDynamic() || element->isStaffText() || element->isTempoText() || element->isSystemText()) {
+                  using staffSegment = QPair<Segment*, int>;
+                  QSet<Segment*> segmentSet;
+                  QSet<staffSegment> staffSet;
+                  for (Element* e : sel.elements()) {
+                        Segment* seg = nullptr;
+                        if (e->isNote()) {
```

Comment:
here and in the next `if'  you're using braces es without need

Reply:
According the coding style:
Exception 2: Use braces also in if-then-else blocks where either the if-code or the else-code covers several lines:
While the "if" and the "else if" are just single line blocks, the final "else" isn't so this exception applies, or am I mistaken?

Reply:
Not sure, it is a minor nitpick anyhow